tests/keras/utils/data_utils_test.py

```diff
@@ -83,6 +85,25 @@ def test_data_utils(in_tmpdir):
 
     origin = urljoin('file://', pathname2url(os.path.abspath('test.tar.gz')))
 
+    path = get_file(dirname, origin, untar=True)
+    filepath = path + '.tar.gz'
+    assert os.path.dirname(os.path.dirname(os.path.abspath(filepath))) \
```

Comment:
Better assign a variable before the `assert` to avoid the `\` linebreak (same below)

Reply:
Ok. Thanks to advice